### PR TITLE
Major Shitmed Bugfixes

### PR DESCRIPTION
**Pull request by gluesniffler**
## About the PR
Fixed some particularly pesky bugs we had laying around. The changelog will display bugs that I have validated are not there anymore, but they might have been fixed previously.

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: Mocho
- fix: (Probably) Fixed all of the slot issues.
- fix: Fixed organ transplants being broken again :trollface:
- fix: You can now perform surgery on changelings.
- fix: Reattached parts now display their state on your status doll correctly.
- fix: Organs now only apply their effects when inserted into the correct slots.
- fix: Now you can put items that are used as surgical tools into disposals again.
- fix: Fixed locational armor not applying to parts. Need to refactor the calculations though.
- fix: Fixed the medical T3 research not showing up, sorry to anyone I might have accidentally gaslit on discord
- tweak: T2 medical tools research now costs 10k research, not sure why I set it to only 5k lol.
- tweak: Added wrapping for alerts to prevent it from overlapping with the body doll. The body status doll isn't an alert yet so expect jankyness with its placement for now. Please report any weirdness so I can tweak values!
- tweak: Changed the boring saw in the surgical duffel bag / crates to a circular saw. 
- tweak: Moved surgery to a verb that comes up when right clicking on a valid entity with a surgical tool in hand.
- tweak: The pen is now usable again as a ghetto surgery tool.

### CODE REVIEW

File: Content.Shared/CCVar/CCVars.cs [hunk 1]
```diff
@@ -2454,7 +2454,7 @@ public static readonly CVarDef<float>
         #region Surgery
 
         public static readonly CVarDef<bool> CanOperateOnSelf =
-            CVarDef.Create("surgery.can_operate_on_self", false, CVar.SERVERONLY);
+            CVarDef.Create("surgery.can_operate_on_self", true, CVar.SERVERONLY);
```
Comment (deltanedas) [inline Content.Shared/CCVar/CCVars.cs:2457]: BASED

File: Resources/Prototypes/Body/Organs/arachnid.yml [hunk 2]
```diff
@@ -164,6 +164,8 @@
   description: "Filters toxins from the bloodstream."
   categories: [ HideSpawnMenu ]
   components:
+  - type: Organ # Shitmed
+    slotId: kidneys
```
[the remark this answers is not in the document]
Comment (deltanedas) [reply Resources/Prototypes/Body/Organs/arachnid.yml:168]: no

File: Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@
   - type: StorageFill
     contents:
       - id: Hemostat
-      - id: Saw
+      - id: SawElectric
```
Comment (deltanedas) [inline Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml]: pro comment

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml (hunk 3 above)
Comment (gluesniffler) [reply Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml]: i forgor

File: Content.Shared/CCVar/CCVars.cs (hunk 1 above)
Comment (gluesniffler) [reply Content.Shared/CCVar/CCVars.cs:2457]: :trollface:

File: Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@
   - type: StorageFill
     contents:
       - id: Hemostat
-      - id: Saw
+      - id: SawElectric # Shitmed Change
```
[the remark this answers is not in the document]
Comment (deltanedas) [reply Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml:10]: fake

**Comment by gluesniffler** [comment]
I think the test fail is unrelated? I havent touched profiles.

**Comment by BombasterDS** [comment]
Pen as ghetto tool won't break hypopen again?

**Comment by gluesniffler** [comment]
> Pen as ghetto tool won't break hypopen again?

no, its a right click verb now

**Review by Piras314 — APPROVED** [review]

**Pull request merged**